dev_env/java_nginx_env/README.md

```diff
@@ -0,0 +1,3 @@
+# Test environment: Java + nginx + uwsgi
+`tar -czvf java_dropin.tar.gz -C /path/to/public_dropin_enironments/java_codegen/ .`
+
```

Comment:
You could probably have more information in this readme

Reply:
I also don't know what the dev_env folder is for.

Reply:
dev_env - temp dev envs to run tests while we haven't switch drop in envs to nginx.

Reply:
so how do you feel about more information

Reply:
I think it is enough: it says what it is: testing environment
and what does it test: nginx_uwsgi

What would you suggest?

Reply:
Updated